keras_cv/models/object_detection/retinanet/retinanet_test.py

```diff
@@ -232,23 +232,34 @@ def test_saved_model(self, save_format, filename):
         self.assertAllClose(model_output, restored_output)
 
 
-@pytest.mark.large
 class RetinaNetSmokeTest(tf.test.TestCase):
     def test_backbone_preset_weight_loading(self):
         # Check that backbone preset weights loaded correctly
         # TODO(lukewood): need to forward pass test once proper weights are
         # implemented
-        keras_cv.models.RetinaNet.from_preset(
-            "resnet50_v2_imagenet",
-            num_classes=20,
-            bounding_box_format="xywh",
-        )
+        weights = ["csp_darknet_tiny_imagenet", "csp_darknet_l_imagenet", 
```

Comment:
Let's load presets without weights here so that we don't have to download all of these weights every CI run.